tests/test___init__.py

```diff
@@ -33,29 +34,29 @@ def test_alexnet_conv2_V4(self):
         model = self.layer_candidate('alexnet', layer='features.5', region='V4', pca_components=1000)
         score = score_model(model_identifier='alexnet-f5-pca_1000', model=model,
                             benchmark_identifier='dicarlo.Majaj2015.V4-pls')
-        assert score.raw.sel(aggregation='center').max() == approx(0.656703, abs=0.005)
+        assert score.raw.sel(aggregation='center').max() == approx(0.633703, abs=0.005)
 
     @pytest.mark.memory_intense
     def test_alexnet_conv5_V4(self):
         model = self.layer_candidate('alexnet', layer='features.12', region='V4', pca_components=1000)
         score = score_model(model_identifier='alexnet-f12-pca_1000', model=model,
                             benchmark_identifier='dicarlo.Majaj2015.V4-pls')
-        assert score.raw.sel(aggregation='center') == approx(0.533175, abs=0.005)
+        assert score.raw.sel(aggregation='center') == approx(0.490769, abs=0.005)
 
     @pytest.mark.memory_intense
     def test_alexnet_conv5_IT(self):
         model = self.layer_candidate('alexnet', layer='features.12', region='IT', pca_components=1000)
         score = score_model(model_identifier='alexnet-f12-pca_1000', model=model,
                             benchmark_identifier='dicarlo.Majaj2015.IT-pls')
-        assert score.raw.sel(aggregation='center') == approx(0.601174, abs=0.005)
+        assert score.raw.sel(aggregation='center') == approx(0.590345, abs=0.005)
 
     @pytest.mark.memory_intense
     def test_alexnet_conv3_IT_mask(self):
         model = self.layer_candidate('alexnet', layer='features.6', region='IT', pca_components=None)
         np.random.seed(123)
         score = score_model(model_identifier='alexnet-f6', model=model,
                             benchmark_identifier='dicarlo.Majaj2015.IT-mask')
-        assert score.raw.sel(aggregation='center') == approx(0.614621, abs=0.005)
+        assert score.raw.sel(aggregation='center') == approx(0.083593, abs=0.005)
```

Comment:
this is a drastic change, seems off

Reply:
This is the result I get :woman_shrugging:

Reply:
Okay since nobody has really optimized the mask benchmarks anyway, it could be the case.